scripts/dts/extract/globals.py

```diff
@@ -351,38 +351,63 @@ def get_binding(node_address):
 def get_binding_compats():
     return bindings_compat
 
+def build_cell_array(prop_array):
+
+    index = 0
+    ret_array = []
+
+    while index < len(prop_array):
+        handle = prop_array[index]
+
+        if handle in {0, -1}:
+            ret_array.append([])
+            index += 1
+        else:
+            # get controller node (referenced via phandle)
+            cell_parent = phandles[handle]
+
+            for k in reduced[cell_parent]['props'].keys():
```

Comment:
Redundant `.keys()`. I wonder if `k` could be given a more descriptive name like `prop_name` too.

Reply:
Removed .keys() changed `k` to `prop`